marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/rest-api/rewriter.xml

```diff
@@ -778,15 +778,15 @@
                 </match-query-param>
                 <match-content-type any-of="text/uri-list">
                     <match-accept any-of="text/uri-list */*">
-                        <dispatch>/data-hub/5/rest-api/endpoints/apply-transform.xqy</dispatch>
+                        <dispatch>/data-hub/4/rest-api/endpoints/apply-transform.xqy</dispatch>
                     </match-accept>
                     <match-header name="accept">
-                        <dispatch>/data-hub/5/rest-api/endpoints/unacceptable-type.xqy</dispatch>
+                        <dispatch>/data-hub/4/rest-api/endpoints/unacceptable-type.xqy</dispatch>
                     </match-header>
-                    <dispatch>/data-hub/5/rest-api/endpoints/apply-transform.xqy</dispatch>
+                    <dispatch>/data-hub/4/rest-api/endpoints/apply-transform.xqy</dispatch>
                 </match-content-type>
             </match-method>
-            <dispatch>/data-hub/5/rest-api/endpoints/apply-transform-bad-request.xqy</dispatch>
+            <dispatch>/data-hub/4/rest-api/endpoints/apply-transform-bad-request.xqy</dispatch>
         </match-path>
```

Comment:
@aebadirad , @akshaysonvane ,
Why are we changing the data-hub/5/rest-api/rewriter.xml to point to 4's modules for default MarkLogic endpoints ? Shouldn't the change be made only for collector (from 5 to 4)  as it is the only legacy custom endpoint, the rest all can be dispatched to 5's modules ?